packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -1113,9 +1113,9 @@ select fn_db_update_config_value('VdcVersion','4.4.0.0','general');
 -- ProductRPMVersion is automatically updated by engine-setup,
 -- but it doesn't harm to keep it up to date here as well.
 select fn_db_update_config_value('ProductRPMVersion','4.4.0.0','general');
-select fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping','apc:secure=secure,port=ipport,slot=port;apc_snmp:port=port,encrypt_options=encrypt_options;bladecenter:secure=secure,port=ipport,slot=port;cisco_ucs:secure=ssl,slot=port;drac5:secure=secure,slot=port;drac7:;eps:slot=port;hpblade:port=port;ilo:secure=ssl,port=ipport;ipmilan:;ilo2:secure=ssl,port=ipport;ilo3:;ilo4:;ilo_ssh:port=port;redfish:port=ipport,secure=ssl;rsa:secure=secure,port=ipport;rsb:;wti:secure=secure,port=ipport,slot=port', '4.2', '4.3');
-select fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping', 'apc:secure=secure,port=ipport,slot=port;apc_snmp:port=port,encrypt_options=encrypt_options;bladecenter:secure=secure,port=ipport,slot=port;cisco_ucs:secure=ssl,slot=port;drac5:secure=secure,slot=port;drac7:;eps:slot=port;hpblade:port=port;ilo:secure=ssl,port=ipport;ipmilan:port=ipport;ilo2:secure=ssl,port=ipport;ilo3:;ilo4:;ilo_ssh:port=port;redfish:port=ipport,secure=ssl;rsa:secure=secure,port=ipport;rsb:;wti:secure=secure,port=ipport,slot=port', '4.4', '4.7');
-select fn_db_update_config_value_for_versions_from_up_to('VdsFenceType', 'apc,apc_snmp,bladecenter,cisco_ucs,drac5,drac7,eps,hpblade,ilo,ilo2,ilo3,ilo4,ilo_ssh,ipmilan,redfish,rsa,rsb,wti','4.2','4.3');
+select fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping','amt_ws:port=ipport;apc:secure=secure,port=ipport,slot=port;apc_snmp:port=port,encrypt_options=encrypt_options;bladecenter:secure=secure,port=ipport,slot=port;cisco_ucs:secure=ssl,slot=port;drac5:secure=secure,slot=port;drac7:;eps:slot=port;hpblade:port=port;ilo:secure=ssl,port=ipport;ipmilan:;ilo2:secure=ssl,port=ipport;ilo3:;ilo4:;ilo_ssh:port=port;redfish:port=ipport,secure=ssl;rsa:secure=secure,port=ipport;rsb:;wti:secure=secure,port=ipport,slot=port', '4.2', '4.3');
+select fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping', 'amt_ws:port=ipport;apc:secure=secure,port=ipport,slot=port;apc_snmp:port=port,encrypt_options=encrypt_options;bladecenter:secure=secure,port=ipport,slot=port;cisco_ucs:secure=ssl,slot=port;drac5:secure=secure,slot=port;drac7:;eps:slot=port;hpblade:port=port;ilo:secure=ssl,port=ipport;ipmilan:port=ipport;ilo2:secure=ssl,port=ipport;ilo3:;ilo4:;ilo_ssh:port=port;redfish:port=ipport,secure=ssl;rsa:secure=secure,port=ipport;rsb:;wti:secure=secure,port=ipport,slot=port', '4.4', '4.7');
+select fn_db_update_config_value_for_versions_from_up_to('VdsFenceType', 'amt_ws,apc,apc_snmp,bladecenter,cisco_ucs,drac5,drac7,eps,hpblade,ilo,ilo2,ilo3,ilo4,ilo_ssh,ipmilan,redfish,rsa,rsb,wti','4.2','4.7');
```

Comment:
it would be simpler if we add it only to cluster level 4.7 but if we add it to all cluster-levels then we don't need to change the 'add' parts of VdsFenceOptionMapping and VdsFenceType

Reply:
I have done the updates in `fn_db_update_config_value...`  functions instead of `fn_db_add_config_value` ones, removed updates for `fn_db_update_config_value_for_versions_from_up_to('VdsFenceOptionMapping'..., '4.2','4.3');` because it is not needed (it updates with the same values as the `fn_db_add_config_value...`). Please take a look :)